dandiapi/api/scheduled_tasks.py

```diff
@@ -0,0 +1,32 @@
+"""
+Define and register any scheduled celery tasks.
+
+This module is imported from celery.py in a post-app-load hook.
+"""
+
+from celery import shared_task
+from celery.schedules import crontab
+from celery.utils.log import get_task_logger
+from django.db.transaction import atomic
+
+from dandiapi.api.models import Version
+from dandiapi.api.tasks import write_manifest_files
+
+logger = get_task_logger(__name__)
+
+
+@shared_task
+@atomic
+def write_draft_manifest_files():
+    logger.info('Writing manifest files for all draft versions')
+    # TODO Optimize this if it causes too much load in production.
```

Comment:
Have you tracked this TODO in your own systems? Out of an abundance of caution (having read the manual page you linked to) I think it would be better if we eventually didn't do this the "wrong" way, so I even think filing an issue just to make this concern global would be a good idea.

Reply:
https://github.com/dandi/dandi-api/issues/530